geniza/common/metadata_export.py

```diff
@@ -78,7 +80,51 @@ def iter_export_data_as_dicts(self):
         # save
         yield from (self.get_export_data_dict(obj) for obj in iterr)
 
-    def iter_export_data_as_csv(self, fn=None, pseudo_buffer=False):
+    def serialize_value(self, value: object) -> str:
+        """A quick serialize method to transform a value into a CSV-friendly string.
+
+        :param value: Any value
+        :type value: object
+
+        :return: Stringified value
+        :rtype: str
+        """
+        if type(value) is bool:
+            return "y" if value else "n"
```

Comment:
could these defined as a lookup on the class? for potential override, similar to separator.  e.g. something like
```suggestion
            return self.true_false[value]
```